settings/common.py

```diff
@@ -286,3 +286,8 @@
 
 # For inviting users to participant and host teams.
 ADMIN_EMAIL = "admin@cloudcv.org"
```

Comment:
There seems to be an error with this email id I think. Can you please change it to `deshrajdry@gmail.com` temporarily till we figure out what is the error?

Reply:
I was able to find out what the problem was. It's working again.